_inc/client/pro-status/index.jsx

```diff
@@ -212,7 +216,7 @@ class ProStatus extends React.Component {
 				}
 			}
 
-			if ( sitePlan.product_slug ) {
+			if ( sitePlan.product_slug && pluginSlug ) {
 				if ( ! hasFree ) {
```

Comment:
this line can be combined with the previous if

Reply:
Addressed in 7def6fde6f0bce1195beac705db37fbead11ce77